docs/core/runtime-config/garbage-collector.md

```diff
@@ -556,9 +556,8 @@ Example:
 
 ## Standalone GC
 
-- Specifies a path to the library containing the garbage collector that the runtime intends to load.
-- For more information, see [Standalone GC loader design](https://github.com/dotnet/runtime/blob/main/docs/design/features/standalone-gc-loading.md).
-
+- Specifies the name of a standalone GC dll that the runtime loads in place of the one it has in the main runtime dll (coreclr.dll). This dll needs to reside in the same directory as coreclr.dll
```

Comment:
```suggestion
- Specifies the name of a standalone GC assembly that the runtime loads in place of the one it has in the main runtime assembly (coreclr.dll). This assembly needs to reside in the same directory as *coreclr.dll*
```

Reply:
I don't think we'd want to use the word assembly here since there are native dlls?

Reply:
Okay. In that case should it be capitalized to "DLL"?

Reply:
it can be. I'm fine either way.

Reply:
Okay, I think it's more correct to be capitalized so I updated my suggestion.